.github/ISSUE_TEMPLATE/Guidance_Issue.md

```diff
@@ -1,9 +1,10 @@
 ---
 name: Community Guidance Request ✨
-about: Suggest somewhere the Windows Terminal Team needs to provide community guidance through new documentation or process.
-title: "Guidance"
+about: Suggest somewhere the Windows Terminal Team needs to provide community guidance
+  through new documentation or process.
+title: ''
 labels: Issue-Docs
-assignees: 'bitcrazed'
```

Comment:
Also don't think I specifically changed default assignees for this template, but GH changed to unquoted.

Reply:
Cinnamon_msft is now default assignee for guidance docs ;)